experimenter/experimenter/jetstream/tasks.py

```diff
@@ -23,7 +23,12 @@ def fetch_experiment_data(experiment_id):
         metrics.incr("fetch_experiment_data.completed")
     except Exception as e:
         metrics.incr("fetch_experiment_data.failed")
-        logger.info(f"Fetching experiment data for {experiment_id} failed: {e}")
+        failure_message = "Fetching experiment data for {experiment_id} "
+        if experiment is not None and hasattr(experiment, "slug"):
+            failure_message += experiment.slug
+            failure_message += " "
+        failure_message += "failed: {e}"
+        logger.info(failure_message)
```

Comment:
nothing wrong on this, just a suggestion 
```python
failure_message = f"Fetching experiment data for {experiment_id} "
if experiment and hasattr(experiment, "slug"):
    failure_message += f"{experiment.slug} "
failure_message += "failed: {e}"
logger.info(failure_message, e=exception)

```

Reply:
Yea it seemed unnecessary to have an f-string just for a space, but I guess it's probably neater that way. I also missed some f-strings so fixed that up in latest commit. Thanks!